source/docs/romi-robot/getting-to-know-romi.rst

```diff
@@ -34,6 +37,32 @@ The wheels have a diameter of 70mm (2.75"). They have a trackwidth of 141mm (5.5
 
 The encoders are connected directly to the motor output shaft and have 12 Counts Per Revolution (CPR). With the provided gear ratio, this nets 1440 counts per wheel revolution.
 
+The motor PWM channels are listed in the table below.
+
++-------------+--------------------------+
+| Channel     | Romi Hardware Component  |
++=============+==========================+
+| PWM 0       | Left Motor               |
++-------------+--------------------------+
+| PWM 1       | Right Motor              |
++-------------+--------------------------+
+
+.. note:: The right motor is hardwired to spin in a backward direction when positive output is applied. Thus, the corresponding motor controller needs to be inverted in robot code.
```

Comment:
hardwired is technically the wrong term here. The right motor is actually wired inverted so they both move forward with the same voltage in the *hardware*. The romi firmware inverts the right motor in code to match standard FRC robot behavior. So perhaps replace it with "configured"

Reply:
The text said "hardwired" before my change, this was not my wording. The old wording left out the important fact that it doesn't matter to the user how it's _wired_, it matters how to _program_ it with WPILib. But, my proposed wording is not precise if the firmware is inverting it.